### PR TITLE
Mccalluc/generated index.json

### PR DESCRIPTION
**Pull request by mccalluc**
## Description

What was changed in this pull request?
- Pull list of fixtures from another json file.
- Confirm that it's up to date

Why is it necessary?
- We want to make sure we run the tests against an up-to-date set of files.

Fixes #___

## Checklist

- [ ] Unit tests added or updated
- [ ] Documentation added or updated
- [ ] Example added or updated
- [ ] Screenshot for visual changes (e.g. new tracks)
- [ ] Updated CHANGELOG.md

### CODE REVIEW

File: travis_build_test.sh [hunk 1]
```diff
@@ -11,6 +11,13 @@ start eslint
 ./node_modules/eslint/bin/eslint.js karma.conf.js app test
 end eslint
 
+start examples
+REGEN_SCRIPT='./docs/examples/regenerate-index.py'
+INDEX_JSON='docs/examples/index.json'
+diff --ignore-space-change <( $REGEN_SCRIPT ) $INDEX_JSON \
```
Comment (pkerpedjiev) [inline travis_build_test.sh]: Sorry, I'm having a hard time wrapping my head around what this is doing. Could you explain it in a sentence or two? Or perhaps add a comment?

Comment (pkerpedjiev) [reply travis_build_test.sh]: It looks like it doesn't actually update index.json but just warns the user if it needs to be updated?

Comment (mccalluc) [reply travis_build_test.sh]: Yep, that's what it does: I'll add a comment. The `<( )` syntax is for process substitution: The output of the command is treated like a file argument that can be read from. (There's also `>( )` for writing.)

Comment (mccalluc) [reply travis_build_test.sh]: Merged recent changes from and develop, and fixed this. I'll go ahead and merge if it comes back green.

**Pull request merged**